### ISSUE TITLE
SplitButton screen reader access issues

### ISSUE THREAD

**Issue by khmakoto** [microsoft/fluentui#21412]
The a11y review of the v9 `SplitButton` component (#20917) identified a bunch of screen reader access issues that need to be fixed:

- `MenuButton` part should be labelled.

**Comment by smhigley** [comment]
One other thing to fix from the review -- Splitbutton shouldn't explicitly set `aria-disabled="false"` for non-disabled buttons. It seems specific to SplitButton, rather than Button.
